src/bin/pg_autoctl/env_utils.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * src/bin/pg_autoctl/env_utils.c
+ *   Utility functions for interacting with environment settings.
+ *
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the PostgreSQL License.
+ *
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+#include "defaults.h"
+#include "env_utils.h"
+#include "log.h"
+
+
+/*
+ * env_empty returns true if the passed environment variable is the empty
+ * string. It returns false when the environment variable is not set or if it
+ * set but is something else than the empty string.
+ */
+bool
+env_empty(const char *name)
```

Comment:
Should the name be `env_found_empty` or something that conveys that the variable is SET to the known value _empty string_ in the environment? when reading the call sites it's not as obvious as I think it would be with the current name.